src/custom/api/gnosisProtocol/api.ts

```diff
@@ -321,6 +321,11 @@ export type ProfileData = {
 
 export async function getProfileData(chainId: ChainId, address: string): Promise<ProfileData | null> {
   console.log(`[api:${API_NAME}] Get profile data for`, chainId, address)
+  return new Promise(function (_resolve, reject) {
```

Comment:
I have forced a rejection to show an Error notification, it will be removed later.